rust/crd/src/lib.rs

```diff
@@ -916,6 +927,7 @@ impl NameNodeConfigFragment {
             },
             logging: product_logging::spec::default_logging(),
             affinity: get_affinity(cluster_name, role),
+            graceful_shutdown_timeout: Some(Duration::from_minutes_unchecked(15)),
```

Comment:
I would like to see this defined as a constant (at the top of the file or in a separate `constants.rs` file (which would be even better)). Same goes for other definitions below.

Reply:
Moved in ce72ff82a3cb70e21ff2061be33295d8f5de5742